UITests/ClearPrivateDataTests.swift

```diff
@@ -13,7 +13,12 @@ class ClearPrivateDataTests: KIFTestCase, UITextFieldDelegate {
         webRoot = SimplePageServer.start()
     }
 
-    override func tearDown() {
+    override func afterEach() {
```

Comment:
Interesting, I didn't know `afterEach` existed. I guess it's equivalent to `tearDown`, but just KIF-friendly?